src/pages/dmt5/_dhero.js

```diff
@@ -74,6 +73,15 @@ const StyledHeader = styled(Header)`
     }
 `
 
+const HeroHeader = styled(Header)`
+    @media (max-width: 1315px) {
+        font-size: 4.8rem;
+    }
+    @media ${device.tablet} {
+        font-size: 36px;
```

Comment:
Can we use `3.6rem` here instead?

Reply:
I don't think we should use `rem` inside media queries

Reply:
I think the article is actually referring to the portion for @media breakpoints and the css styling bit won't be affected. The reason I actually used `px` here is because our root font sizes changes across different screen widths and `3.6rem` won't result in `36px` here.